Sources/Drain.swift

```diff
@@ -33,9 +34,6 @@ public final class Drain: DataRepresentable, Stream {
 
     public init(_ buffer: Data) {
         self.buffer = buffer
-        if buffer.bytes.isEmpty {
-            close()
-        }
```

Comment:
@paulofaria that's because i'm manipulating the drain in the test, though. You shouldn't be able to open a drain once it's closed